project-codeflare/codeflare-sdk
# [WIP] Adding datasets to dependencies

# Issue link
[RHOAIENG-6475](https://issues.redhat.com/browse/RHOAIENG-6475)

# What changes have been made
When running the training job in the basic_interactive demo notebook the following error is encountered:

`ModuleNotFoundError: No module named 'datasets'`

This dependency has been added to the pyproject.toml file.


# Verification steps
- In a jupyter notebook install codeflare SDK from this PR
```
git clone https://github.com/Fiona-Waters/codeflare-sdk.git -b add-datasets
pip install poetry
poetry install
```
- Restart the kernal and run the basic_interactive demo from start to finish and ensure you do not see the error.

## Checks
- Testing Strategy
